etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/messages/SendMessageHelper.java

```diff
@@ -21,67 +21,66 @@ public static SendMessageHelper getInstance() {
 
     private SendMessageHelper() {}
 
-    public void savePartnerMetadataForReceivedMessage(PartnerMetadata partnerMetadata) {
-        if (partnerMetadata.receivedSubmissionId() == null) {
+    public void savePartnerMetadataForInboundMessage(PartnerMetadata partnerMetadata) {
+        if (partnerMetadata.inboundReportId() == null) {
             logger.logWarning(
-                    "Received submissionId is null so not saving metadata for received message");
+                    "Inbound reportId is null so not saving metadata for received message");
             return;
         }
         try {
-            partnerMetadataOrchestrator.updateMetadataForReceivedMessage(partnerMetadata);
+            partnerMetadataOrchestrator.updateMetadataForInboundMessage(partnerMetadata);
         } catch (PartnerMetadataException e) {
             logger.logError(
-                    "Unable to save metadata for receivedSubmissionId "
-                            + partnerMetadata.receivedSubmissionId(),
+                    "Unable to save metadata for inboundReportId "
+                            + partnerMetadata.inboundReportId(),
                     e);
         }
     }
 
-    public void saveSentMessageSubmissionId(String receivedSubmissionId, String sentSubmissionId) {
-        if (sentSubmissionId == null || receivedSubmissionId == null) {
+    public void saveReportIds(String inboundReportId, String outboundReportId) {
+        if (outboundReportId == null || inboundReportId == null) {
             logger.logWarning(
-                    "Received and/or sent submissionId is null so not saving metadata for sent result");
+                    "Inbound and/or outbound reportId is null so not saving metadata for sent result");
             return;
         }
 
         try {
-            partnerMetadataOrchestrator.updateMetadataForSentMessage(
-                    receivedSubmissionId, sentSubmissionId);
+            partnerMetadataOrchestrator.updateMetadataForOutboundMessage(
+                    inboundReportId, outboundReportId);
         } catch (PartnerMetadataException e) {
             logger.logError(
-                    "Unable to update metadata for received submissionId "
-                            + receivedSubmissionId
-                            + " and sent submissionId "
-                            + sentSubmissionId,
+                    "Unable to update metadata for inbound reportId "
+                            + inboundReportId
+                            + " and sent outbound reportId "
```

Comment:
You can rename `sent outbound reportId` => `outboundReportId`

Reply:
True. Removed that